### PR TITLE
s/LoadMap/LoadState in node_grpc_main.cc

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc [hunk 1]
```diff
@@ -57,7 +57,7 @@ void Run() {
   Node node(node_options, std::move(map_builder), &tf_buffer);
 
   if (!FLAGS_map_filename.empty()) {
```
Comment (ojura) [inline cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc]: Rename the flag as well, to be consistent with the offline and online node?

Comment (ojura) [reply cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc]: Or, maybe we can delay that rename once GRPC supports loading unfrozen trajectories as well.

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**